docker/helm-chart/values-minikube.yaml

```diff
@@ -0,0 +1,226 @@
+---
+config:
```

Comment:
Freqtrade uses config files in .json format. 

i don't really see how this fits into the picture. Aside from that, it seems way too opinionated.